src/graphnet/pisa/utils.py

```diff
@@ -0,0 +1,586 @@
+import numpy as np
```

Comment:
I would rename this file `utils.py` -> `contours.py` to better signify its contents. Perhaps preferably, split it up into `fit.py` and `contour.py` to make the file sizes more manageable.

Reply:
I have renamed utils.py -> fitting.py